src/java.net.http/share/classes/jdk/internal/net/http/HttpClientImpl.java

```diff
@@ -1690,18 +1694,24 @@ BufferSupplier getSSLBufferSupplier() {
         return sslBufferSupplier;
     }
 
-    private Duration getIdleConnectionTimeoutProp() {
-        String s = Utils.getNetProperty("jdk.httpclient.keepalive.timeout");
-        if (s != null) {
-            try {
-                long val = Long.parseLong(s);
-                if (val >= 0)
-                    return Duration.ofSeconds(Long.parseLong(s));
-            } catch (NumberFormatException ignored) {}
-        }
+    private Duration getIdleConnectionTimeout() {
+        if (IDLE_CONNECTION_TIMEOUT >= 0)
+            return Duration.ofSeconds(IDLE_CONNECTION_TIMEOUT);
         return null;
     }
 
+    private static long getTimeoutProp(String prop, long def) {
+        String s = Utils.getNetProperty(prop);
+        try {
+            if (s != null) {
+                long timeoutVal = Long.parseLong(s);
+                System.err.println(timeoutVal);
```

Comment:
Stray println? You could use `Log.logTrace` here to print the value.

Reply:
Gah, thought I zapped all of these! Thanks for spotting. I think there is no need to log anything here, so I will just remove the println.